src/Etsy/methods.json

```diff
@@ -759,9 +759,9 @@
     "params": {
       "listing_id": "int",
       "products": "stringJSON",
-      "price_on_property": "array(int)",
-      "quantity_on_property": "array(int)",
-      "sku_on_property": "array(int)"
+      "price_on_property": "string",
```

Comment:
We can not do this, because this file is downloaded and used as it is from Etsy.